Makefile

```diff
@@ -173,7 +173,7 @@ export TEST_OPS ?= ""
 test: test-no-verify-changes verify-no-changes ## Run tests and verify no changes
 
 .PHONY: test-no-verify-changes
-test-no-verify-changes: manifests generate go-verify fmt vet test-imports envtest ## Generate and format code, run tests, generate manifests and bundle
+test-no-verify-changes: go-verify generate fmt vet test-imports envtest ## Generate and format code, run tests, generate manifests and bundle
```

Comment:
why dropping `manifests`?

Reply:
I had an issue with downgrading `controller-gen`, and I suspected it was due to the `manifests` target, but it was with old local files under `/bin`.